builtins/safe-default-configuration.txt

```diff
@@ -169,3 +169,65 @@ th
 - lang
 - title
 
+// MathML, per https://github.com/WICG/sanitizer-api/issues/103#issuecomment-2591328351
+
+math math
+math merror
+math mfrac
+math mi
+math mmultiscripts
+math mn
+math mo
+- form
+- fence
+- separator
+- lspace
+- rspace
+- stretchy
+- symmetric
+- maxsize
+- minsize
+- largeop
+- movablelimits
+math mover
+- accent
+math mpadded
+- width
+- height
+- depth
+- lspace
+- voffset
+math mprescripts
+math mroot
+math mrow
+math ms
+math mspace
+- width
+- height
+- depth
+math msqrt
+math mstyle
+math msub
+math msubsup
+math msup
+math mtable
+math mtd
+- columnspan
+- rowspan
+math mtext
+math mtr
+math munder
+- accentunder
+math munderover
+- accent
+- accentunder
+math semantics
+
+[MathML Global]
```

Comment:
Maybe this should have a comment "Selection of attributes. Most are purposely omitted." similar to HTML?

and a link to https://w3c.github.io/mathml-core/#global-attributes

Reply:
Done.